pub_topic: "/output"
x_value: 4.0
y_value: 3.1415926535897932384626433832795028841971
z_value: 900.0